net/knot-resolver/files/kresd.init

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh /etc/rc.common
+
+. /usr/share/resolver/functions.sh
+
+START=61
+STOP=00
+
+USE_PROCD=1
+PROG=/usr/sbin/kresd
+CONFIGFILE=/tmp/kresd/kresd.config
+ROOTKEYFILE=/etc/knot-resolver/root.keys
+DEFAULT_RUNDIR=/tmp/kresd
+USERNAME=kresd
+GROUP=kresd
+
+service_triggers() {
+	procd_add_reload_trigger "dhcp"
+}
+
+init_header() {
+	echo "--Automatically generated file; DO NOT EDIT" > $CONFIGFILE
+	echo "modules = {" >> $CONFIGFILE
+	echo "    'hints > iterate'"  >> $CONFIGFILE
+	echo "  , 'policy'" >> $CONFIGFILE
+	if [ "$prefetch" \!= 0 ]; then
+		echo "  , 'stats'" >> $CONFIGFILE
+		echo "  , predict = {" >> $CONFIGFILE
+		echo "        window = 30 -- 30 minutes sampling window" >> $CONFIGFILE
+		echo "      , period = 24*(60/30) -- track last 24 hours" >> $CONFIGFILE
+		echo "  }" >> $CONFIGFILE
+	fi
+	echo "}" >> $CONFIGFILE
+	echo "hints.use_nodata(true)" >> $CONFIGFILE
+}
+
+init_rootkey() {
+	echo "trust_anchors.remove('.')" >> $CONFIGFILE
+	echo "trust_anchors.add_file('$ROOTKEYFILE', true)" >> $CONFIGFILE
+}
+
+modify_rundir() {
+	local rundir
+
+	#create rundir
+	rundir="$DEFAULT_RUNDIR"
+
+	if [ ! -d "$rundir" ]; then
+		mkdir -p "$rundir"
+	fi
+
+	if id "$1" >/dev/null 2>&1; then
+		chown -R $USERNAME:$GROUP "$rundir"
+	fi
+}
+
+start_service() {
+	procd_open_instance
+	procd_set_param file /etc/config/resolver
+	procd_set_param command "$PROG"
+	procd_append_param command --noninteractive
+	procd_set_param respawn
+	modify_rundir
+	init_header
+	init_rootkey
+
+	# knot-resolver config
+	procd_append_param command -c "$CONFIGFILE"
+	procd_append_param command -a "0.0.0.0#53"
+	procd_append_param command -a "::0#53"
+	procd_close_instance
+}
+
+reload_service()
+{
```

Comment:
Why this section

Reply:
I removed that. It was leftover from some experiments with static domains from dhcp config.